go/enclave/enclave.go

```diff
@@ -71,6 +73,8 @@ type enclaveImpl struct {
 	transactionBlobCrypto crypto.TransactionBlobCrypto
 	profiler              *profiler.Profiler
 	logger                gethlog.Logger
+
+	stopInterrupt *int32
```

Comment:
I don't think we need to guard every rpc method with this, the rpc server has a stop method that should be called when the enclave shuts down so that no methods can be called at all.